templates/10-opcache.ini.j2

```diff
@@ -0,0 +1,95 @@
+; Enable Zend OPcache extension module
+zend_extension=opcache.so
+
+; Determines if Zend OPCache is enabled
+opcache.enable={{ d7_opcache_enable }}
+
+; Determines if Zend OPCache is enabled for the CLI version of PHP
+;opcache.enable_cli=0
```

Comment:
We should see what we're doing for apc on php cli caching, I immediately thought about caching stuff for drush bootstrap. Can't remember if we are doing that at oulib.